src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs

```diff
@@ -295,7 +295,7 @@ public static Level1DynamicFields Parse(string message, DynamicFieldset[] fields
             int tick = default;
             int tickID = default;
             int totalVolume = default;
-            string type = default;
+            string type = "Q";
```

Comment:
so this will only be Q update ?

Reply:
I goofed.  It should also return P.  See comments above. this does not work properly.